api/groups/blockGroup_test.go

```diff
@@ -454,26 +462,24 @@ func TestGetAlteredAccountsByNonce_ShouldWork(t *testing.T) {
 
 	ws := startWebServer(blockGroup, "block", getBlockRoutesConfig())
 
-	response, code := httpGetBlock(ws, "/block/altered-accounts/by-nonce/37")
+	response, code := httpGetAlteredAccountsForBlockBlock(ws, "/block/altered-accounts/by-nonce/37")
 	require.Equal(t, http.StatusOK, code)
-	require.NotNil(t, response)
+	require.Equal(t, expectedResponse, response.Data.Accounts)
```

Comment:
[comment]

Perhaps we could also check here and below that error field from `response` is empty